### DISCUSSION TITLE
Can I set a default value based on a role?

### DISCUSSION THREAD

**Discussion by abdonrd** [directus/directus#11738] [Archive]
Right now, the fields have an option to set the default value in the Schema:

<img width="855" alt="Screenshot 2022-02-19 at 22 05 09" src="https://user-images.githubusercontent.com/1007051/154819069-5ae4e558-7080-43b1-b4e6-6b91ada2cbfe.png">

But there is any option to set a different default value based on the user role?

Thanks!

**Comment by benhaynes** [comment]
Yes, under **Settings > Permissions > Role > Create**

**Reply by abdonrd** [reply]
Thanks @benhaynes! 👏🏻 

I hadn't found it, because I was trying to do it on the Administrator role. With this role it is not possible, right?

**Reply by rijkvanzanten** [reply]
> With this role it is not possible, right?

That's correct, administrators don't have any permissions (and therefore no presets)

**Reply by abdonrd** [reply]
Thanks both!

**Reply by AxeemHaider** [reply]
Not understand how you are setting default value based on user role?

**Reply by jbeckton** [reply]
@benhaynes  do you have a Docs link for this? I see a text area for the field presets but cannot find any info on how to use it. I'm assuming it's a similar format to the filter rules.

**Reply by benhaynes** [reply]
@estheragbaje ... can you assist here?

**Reply by jbeckton** [reply]
I figured it out, it just takes a json object in the shape of the collection fields. Thanks

**Reply by benhaynes** [reply]
I'm glad! I still want @estheragbaje to ensure this feature is covered in the Docs. 👍

**Reply by estheragbaje** [reply]
Sure, I'd work on adding this to the docs @benhaynes

**Comment by jbeckton** [comment]
@abdonrd I don't know if it's a best practice or not as I am just getting started with Directus but you could extend the built in directus_role collection with a new field to hold the value based on the role and access the dynamic variables @Current_Role.my_field inside filter rules / presets
